server/data/deliusClient.js

```diff
@@ -0,0 +1,47 @@
+const superagent = require('superagent')
+const logger = require('../../log')
+const config = require('../config')
+const { unauthorisedError } = require('../utils/errors')
+
+const timeoutSpec = {
+  response: config.nomis.timeout.response,
+  deadline: config.nomis.timeout.deadline,
+}
+
+const { apiUrl } = config.delius
+
+module.exports = signInService => {
+  return {
+    getROPrisoners(deliusStaffCode) {
+      const path = `${apiUrl}/staff/staffCode/${deliusStaffCode}/managedOffenders`
+      const query = { current: true }
+      return deliusGet({ path, query })
```

Comment:
The query parameter ?current=true is not required as this is added within the community proxy. Alternatively, I could rework the query parameters in the proxy?

Reply:
Done